blog-meeting-minutes/2024-02-29-security-hour.md

```diff
@@ -0,0 +1,30 @@
+---
+slug: security-office-hour-2024-02-29
+title: Security Office Hour 29.02.2024
+authors: 
+    - daniel_dilger
+tags: [meeting-minutes, community, security]
```

Comment:
Is `security` the related tag now we discussed last friday? Can this be used to filter for all security notes?